src/api/app/assets/stylesheets/webui2/comments.scss

```diff
@@ -1,6 +1,16 @@
 #comments {
   .media .media-body {
-    hyphens: auto;
+
+    // hyphens isn't supported on Desktop chrome
+    // https://caniuse.com/#search=hyphens
+    @supports(hyphens:auto) {
+      hyphens: auto;
+    }
+    @supports not(hyphens: auto) {
+      // break-word is depreacted. But works on Chrome well
+      // https://developer.mozilla.org/en-US/docs/Web/CSS/word-break
+      word-break: break-word;
```

Comment:
As we discussed together, this is a gamble. Will Chrome support `hyphens: auto` before `word-break: break-word` is not supported anymore? I prefer to find another solution.

Reply:
Hi @dmarcoux 

Gambling is one way to see it. I see it as reducing the numbers of devices which won' t be able to see the page correctly to the minimum as possible.

What would be the alternative? Stop to use `hyphens: auto` since it's not supported by all major browsers and stick with the `break-all` and show it badly in all browsers? Not sure if it's better.

Reply:
And maybe we can find the long term solution **after** we have fixed the page for people? :cowboy_hat_face:

Reply:
I agree with that @hennevogel

Reply:
I would use `break-all` for Chrome and if Chrome users complain, well, they can use Firefox loll

Reply:
Let's re-open https://github.com/openSUSE/open-build-service/issues/7796

Reply:
I don't see a problem to use temporarily `word-break: break-word` if that's the best technical solution.. Following your reasoning, because we have the `support` guard, if Chrome stops to support the `break-word` before it supports fully `hyphens`, then the user can still move to Firefox anyway..